src/LiveDevelopment/LiveDevServerManager.js

```diff
@@ -88,6 +89,7 @@ define(function (require, exports, module) {
      *  particular url. Providers that register with a higher priority will
      *  have the opportunity to provide a given url before those with a
      *  lower priority. The higher the number, the higher the priority.
+     * @return {{create: function():BaseServer, priority: number}}
```

Comment:
Since it's only used as an opaque handle, no need to specify it's internals. should be just `{object}`.